ast/ddl.go

```diff
@@ -363,7 +364,19 @@ type IndexOption struct {
 
 // Restore implements Node interface.
 func (n *IndexOption) Restore(ctx *RestoreCtx) error {
-	return errors.New("Not implemented")
+	if n.KeyBlockSize > 0 {
+		ctx.WriteKeyWord(" KEY_BLOCK_SIZE = ")
```

Comment:
As explained in https://github.com/pingcap/parser/pull/81#discussion_r242415210, we expect the output of `Restore()` is a trimmed string (no leading or trailing spaces).